R/aqi_equation.R

```diff
@@ -0,0 +1,87 @@
+
+#' @title Calculate Air Quality Index (AQI) and classification
+#'
+#' @param pollutant Either "pm2.5" or "ozone"
+#' @param concentration A vector of concentrations of that pollutant,
+#' in ug/m^3 for pm2.5 or in ppb for ozone
+#'
+#' @return A data frame with the columns Concentration, AQI, Class, and Color
+#' @export
+#'
+#' @examples
+#'
+#'
+
+#### Set up reference tables:
+
+### PM2.5:
+
+PM25.Class<- c("Good", "Moderate", "Unhealthy for Sensitive Groups",
```

Comment:
If these values are permanent, you can move them inside the function. Later on we can either create a global variable or move them into another function.